### PR TITLE
[coreml] Update for Xcode 12 beta 2

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

File: src/coreml.cs [hunk 1]
```diff
@@ -968,6 +985,18 @@ interface MLUpdateTask {
 		[return: NullAllowed]
 		MLUpdateTask Create (NSUrl modelUrl, IMLBatchProvider trainingData, [NullAllowed] MLModelConfiguration configuration, MLUpdateProgressHandlers progressHandlers, [NullAllowed] out NSError error);
 
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Static]
+		[Export ("updateTaskForModelAtURL:trainingData:completionHandler:error:")]
+		[return: NullAllowed]
+		MLUpdateTask Create (NSUrl modelUrl, IMLBatchProvider trainingData, Action<MLUpdateContext> completionHandler, [NullAllowed] out NSError error);
```
Comment (whitneyschmidt) [inline src/coreml.cs:992]: `[Async]` for completion handler?

File: src/coreml.cs [hunk 2]
```diff
@@ -968,6 +985,18 @@ interface MLUpdateTask {
...
+
+		[Watch (7,0), TV (14,0), Mac (10,16), iOS (14,0)]
+		[Static]
+		[Export ("updateTaskForModelAtURL:trainingData:progressHandlers:error:")]
+		[return: NullAllowed]
```
Comment (whitneyschmidt) [inline src/coreml.cs:997]: '[Async]`?

File: src/coreml.cs (hunk 1 above)
Comment (spouliot) [reply src/coreml.cs:992]: `[Async]` only works when the completion handler is the last argument

File: src/coreml.cs (hunk 2 above)
Comment (spouliot) [reply src/coreml.cs:997]: progress != completion
you don't want to wait on something that's called multiple times

**Pull request merged**